client/src/App.jsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import axios from 'axios';
+import styled from 'styled-components';
 import PhotosContainer from './PhotosContainer';
 
+const StyledContainer = styled.div`
+  display: grid;
+  grid-template-columns: 50% 25% 25%;
+  grid-template-rows: 50% 50%;
+  height: 475px;
+  width: 100%;
+  background: #000;
+  overflow: hidden;
+`;
+
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { photos: [], isContainerHovered: false };
 
-    this.state = { photos: [] };
+    this.onContainerMouseOver = this.onContainerMouseOver.bind(this);
+    this.onContainerMouseOut = this.onContainerMouseOut.bind(this);
   }
 
   componentDidMount() {
-    axios.get('/photos/?listingid=2')
+    axios.get('/photos/?listingid=7')
```

Comment:
Good to hardcode for now, but eventually you'll have to make it take id from the url you type in the browser, the code to get the id number is: window.location.href.match(/listingid\s*=\s*(.*)/)

Reply:
Yeah, it's on my list of things to complete tomorrow. Thanks!